engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -39,6 +39,23 @@ <h4 translate> Basic Information </h4>
         </select>
       </div>
 
+      <div
+        bst-form-group label="{{ 'Required Tags' | translate }}"
```

Comment:
I'm having a premonition .... we are receiving feedback to change what this is called ... maybe we should get buy in from PM on what the call this in the user-facing sense

Reply:
yes, wording is difficult as always!  Also looking for feedback on the API description

Reply:
@jturel commented on Aha.